### PR TITLE
Support themes in Hints

### PR DESCRIPTION
**Pull request by AlexanderPrendota**
Hello!

This PR for using different themes in hints.

### CODE REVIEW

**Comment by AlexanderPrendota** [comment]
So, if CodeMirror can not support "`" I can replace to ```var theme = "CodeMirror-Hints " + theme;```

File: addon/hint/show-hint.js [hunk 1]
```diff
@@ -200,7 +200,8 @@
     var widget = this, cm = completion.cm;
 
     var hints = this.hints = document.createElement("ul");
-    hints.className = "CodeMirror-hints";
+    var theme = completion.cm.options.theme;
+    hints.className = `CodeMirror-hints ${theme}`;
```
Comment (marijnh) [inline addon/hint/show-hint.js]: The addons should be ES5, so you can't use template literals here.

Comment (AlexanderPrendota) [reply addon/hint/show-hint.js]: @marijnh okay. I'll change to 
`var theme = "CodeMirror-Hints " + theme;`

**Comment by AlexanderPrendota** [comment]
@marijnh Element.classList is not supported some browsers that's why I use plain string concatenation.

**Comment by AlexanderPrendota** [comment]
@marijnh Yes. Exactly. Right now all is fine. I tested it. =) Thanks!

**Comment by marijnh** [comment]
Great. Merged as 70408ced

**Pull request closed without merging**